homeassistant/components/lutron_caseta/__init__.py

```diff
@@ -255,6 +259,13 @@ def _async_register_button_devices(
     return button_devices_by_dr_id, device_info_by_device_id
 
 
+def _handle_none_keypad_serial(keypad_device: dict, bridge_serial: int) -> str:
+    return (
+        keypad_device["serial"]
+        or f"{str(bridge_serial)}_{str(keypad_device['device_id'])}"
```

Comment:
You can drop the str() as the f string conversion will do it for you

Reply:
done